.github/workflows/ci-integration-tests.yml

```diff
@@ -68,29 +71,31 @@ jobs:
     - name: Run the Agama smoke test
       run:  podman exec agama curl http://localhost
 
-    - name: Check Playwright version
-      run:  podman exec agama playwright --version
+    - name: Run the Puppeteer tests
+      # update the test file and the runner script from git
+      run:  podman exec agama bash -c
+            "cp /checkout/puppeteer/tests/test_root_password.js /usr/share/agama/integration-tests/tests &&
+            cp /checkout/puppeteer/agama-integration-tests /usr/bin/agama-integration-tests"
 
-    - name: Run the Playwright tests
-      # user authentication is not required when cockpit runs a local session
-      # run the tests in the Chromium browser
-      run:  podman exec agama bash -c "cd /checkout/playwright && SKIP_LOGIN=true playwright test --trace on --project chromium"
+    - name: Run the Puppeteer tests
+      # run the test
+      run:  podman exec agama bash -c "cd /var/log/puppeteer &&
```

Comment:
this cd looks kind of strange. should not agama-integration-tests script ensure that artifacts will be in correct place itself?

Reply:
The script itself saves the log to `./log/` so it's up to the caller where the logs will be saved.

In this case we use `/var/log` as it is already used for the YaST logs, but it could be anywhere else.